content/sd/0003-tempo-e-sincronizacao.md

```diff
@@ -170,6 +180,36 @@ servidor de tempo falhar, o sistema não consegue sincronizar os relógios.
 Cristian propõe que o pedido seja feito em _multicast_ a vários servidores de
 tempo, selecionando o que responder primeiro, resultando na melhor precisão.
 
+:::info[Exercício]
+
+Um cliente pretende sincronizar-se com um servidor e para tal regista o RTT e os tempos
+enviados pelo servidor:
+
+| RTT (ms)            | Time (hh\:mm\:ss)             |
+| ------------------- | ----------------------------- |
+| <center>22</center> | <center>10:54:23.674</center> |
+| <center>25</center> | <center>10:54:25.450</center> |
+| <center>20</center> | <center>10:54:28.342</center> |
```

Comment:
No need for `<center>` tags, [markdown tables support alignment per column](https://github.github.com/gfm/#tables-extension-).
```suggestion
| RTT (ms) | Time (hh\:mm\:ss) |
| :------: | :---------------: |
|    22    |   10:54:23.674    |
|    25    |   10:54:25.450    |
|    20    |   10:54:28.342    |
```